config/database.yml

```diff
@@ -81,5 +85,3 @@ test:
 production:
   <<: *default
   database: com-prototype_production
-  username: com-prototype
```

Comment:
My intention here is to cater for port connection setups. This only working for socket connection setup